docs/minio.md

```diff
@@ -11,341 +11,305 @@ sidebar_label: MinIO
 
-The steps for the creation of StorageClass can be found below:
+### Select OpenEBS storage engine
 
-- [cStor](https://docs.openebs.io/docs/next/ugcstor.html#creating-cStor-storage-class)
-- [Local PV hostpath](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html), [Local PV device](https://docs.openebs.io/docs/next/uglocalpv-device.html), [Local PV with Customized hostpath](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html#create-storageclass)
-- [Jiva](https://docs.openebs.io/docs/next/jivaguide.html#create-a-sc)
+A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and chooses the right engine that suits your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](https://docs.openebs.io/docs/next/overview.html#openebs-storage-engines).
 
+In this document, it is mentioned about the installation of MinIO operator using OpenEBS Local PV device. 
 
-<h3><a class="anchor" aria-hidden="true" id="install-openebs"></a>Launch MinIO application by using the corresponding StorageClass</h3>
+### Configure OpenEBS Local PV StorageClass
 
-<h4><a class="anchor" aria-hidden="true" id="minio-localpv"></a>MinIO on Local PV</h4>
-
-MinIO can be provisioned on OpenEBS Local PV either on hostpath or on block Device. Local PV can be used if low-latency is a must requirement, and if the application can handle the replication by itself in `distributed mode` and/or application does not require replication at all in `standalone mode`.
-
-**In Standalone mode:**
-
-Local PV can be provisioned by following of the options below:
-
-- Hostpath-based
-
-  In this case, Local PV volume will be provisioned on the hostpath created on the node where the application has been scheduled. A persistent volume (PV) will be created with a specified size on `/var/openebs/local` directory on the same node when it uses default StorageClass  `openebs-hostpath`. 
-  Customizing the basepath can also be done using the steps provided [here](https://docs.openebs.io/docs/next/uglocalpv.html#configure-hostpath). Using the customized basepath, user can mount the device and use this mounted directory into the basepath field in the StorageClass.
+There are 2 ways to use OpenEBS Local PV.
 
+- `openebs-hostpath` - Using this option, it will create Kubernetes Persistent Volumes that will store the data into OS host path directory at: /var/openebs/<cassandra-pv>/. Select this option, if you don’t have any additional block devices attached to Kubernetes nodes. You would like to customize the directory where data will be saved, create a new OpenEBS Local PV storage class using these [instructions](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html#create-storageclass). 
```

Comment:
This needs to be changed to reflect minio 
"directory at: /var/openebs/<cassandra-pv>/"

Reply:
Done